<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupwindow</class>
 <widget class="QDialog" name="signupwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="firstNameLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>460</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;   /* dark blue */
    color: black;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 21px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
color: white;
}</string>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>590</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 21px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
   </property>
   <property name="text">
    <string>signup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>15</red>
         <green>106</green>
         <blue>186</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>89</red>
         <green>139</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>15</red>
         <green>106</green>
         <blue>186</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>89</red>
         <green>139</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Your Account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>National ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>1181</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/screen1Background/screen1Background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>530</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>238</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>540</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;   /* dark blue */
    color: black;
    border: none;                /* no border line */
    border-radius: 10px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 14px;
    background-image: none;      /* no image */
}</string>
   </property>
   <property name="text">
    <string>SHOW</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>firstNameLine</zorder>
  <zorder>phoneNumLine</zorder>
  <zorder>loginButton</zorder>
  <zorder>signUpButton</zorder>
  <zorder>label</zorder>
  <zorder>userNameLine</zorder>
  <zorder>idLine</zorder>
  <zorder>emailLine</zorder>
  <zorder>passwordLine</zorder>
  <zorder>lastNameLine</zorder>
  <zorder>showButton</zorder>
 </widget>
 <tabstops>
  <tabstop>firstNameLine</tabstop>
  <tabstop>lastNameLine</tabstop>
  <tabstop>userNameLine</tabstop>
  <tabstop>emailLine</tabstop>
  <tabstop>idLine</tabstop>
  <tabstop>phoneNumLine</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>passwordLine</tabstop>
  <tabstop>signUpButton</tabstop>
  <tabstop>showButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
